inaugurator/storagedevices.py

```diff
@@ -54,7 +54,15 @@ def readSmartDataFromAllDevices(cls, talkToServer=None, failOnFailedHealthTest=F
                     if failOnFailedHealthTest:
                         if talkToServer is not None:
                             talkToServer.healthTestFailed(device)
-                        raise ex
+
+                        # check if device exist in failOnDevices
+                        if any([(device in cur_dev) for cur_dev in failOnDevices]):
```

Comment:
Why not
```suggestion
                        if device in failOnDevices:
```

Reply:
So please add a comment and consider documenting it in a docstring

Reply:
added

Reply:
added